src/main/webapp/resources/js/components/AnalysesQueue.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from "react";
+import { Tag } from "antd";
+import { fetchAnalysesQueueCounts } from "./../apis/analysis/analysis";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faRunning } from "@fortawesome/free-solid-svg-icons";
+import { SPACE_XS } from "../styles/spacing";
+import styled from "styled-components";
+import { formatNumber } from "../utilities/number-utilities";
+import { blue4 } from "../styles/colors";
+
+const LabelTD = styled.td`
+  font-weight: bold;
+`;
+
+const ValueTD = styled.td`
+  width: 50px;
+  text-align: right;
+  font-family: monospace;
+`;
+
+/**
+ * React component for rendering the current server status
+ * for running analyses.
+ * @return {*}
+ * @constructor
+ */
+export function AnalysesQueue({}) {
+  const [running, setRunning] = useState(0);
+  const [queued, setQueued] = useState(0);
+
+  useEffect(() => {
+    fetchAnalysesQueueCounts().then(data => {
+      setRunning(data.running);
+      setQueued(data.queued);
+    });
+  }, [fetchAnalysesQueueCounts]);
+
+  return (
+    <Tag color={"blue"}>
+      <div
+        style={{
+          width: 150,
+          display: "flex",
+          justifyContent: "space-between",
+          alignContent: "center"
+        }}
+      >
+        <FontAwesomeIcon
```

Comment:
I like the idea of the running man icon, but I don't think it's clear enough.  Any ideas for text replacements?  Maybe a tooltip? I might even say it looks better with no icon or text.

Reply:
Updated documentation and add hover tooltip.